devimint/src/util.rs

```diff
@@ -79,10 +82,19 @@ impl ProcessManager {
             .cmd
             .spawn()
             .with_context(|| format!("Could not spawn: {name}"))?;
-        Ok(ProcessHandle(Arc::new(ProcessHandleInner {
+        let handle = ProcessHandle(Arc::new(Mutex::new(ProcessHandleInner {
             name: name.to_owned(),
             child: Some(child),
-        })))
+        })));
+        self.handles.lock().await.push(handle.clone());
+        Ok(handle)
+    }
+
+    pub async fn kill_all_children(&self) -> Result<()> {
+        for handle in self.handles.lock().await.iter() {
+            handle.kill().await?;
```

Comment:
This terminating early could prevent us from killing other children. Biggest issue I see.

Reply:
Now wil kill everything in parallel and just log the resulting errors